test.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Comment
+ */
+
+echo 'Hello World!';
+
+$a = function_call();
+
+
+class TestFile {
```

Comment:
🚫 **[phpcs]** <Squiz.Commenting.ClassComment.Missing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing doc comment for class TestFile